### PR TITLE
Permissions Issue: Accepted Senders Only has Delegated Permissions. 

### PR DESCRIPTION
**Pull request by georgend**
Add Groups["{guid}"].AcceptedSenders to list of Delegated permission endpoints since endpoint does not support Application Permissions.
- List Accepted Senders - https://docs.microsoft.com/en-us/graph/api/group-list-conversations?view=graph-rest-1.0&tabs=http
image

***Minor Changes***
Add various compilations options.
- Deterministic Compilation
- Speed Up Startup using Tiered Compilation Options - https://docs.microsoft.com/en-us/dotnet/core/run-time-config/compilation
- Emit Embedded Pdbs when Compiling Snippets.

### CODE REVIEW

File: msgraph-sdk-raptor-compiler-lib/Models/TestsSetup.cs [hunk 1]
```diff
@@ -27,5 +27,28 @@ public static IPublicClientApplication SetupPublicClientApp(RaptorConfig config)
...
+                .GetResult();
+            foreach (var account in accounts)
+            {
+                publicClientApplication.RemoveAsync(account)
+                    .GetAwaiter()
+                    .GetResult();
+            }
+        }
+
+        public static void CleanupPublicClientApp(IConfidentialClientApplication confidentialClientApplication)
```
Comment (MIchaelMainer) [inline msgraph-sdk-raptor-compiler-lib/Models/TestsSetup.cs:44]: Likely meant for this to be called `CleanupConfidentialClientApp`

**Review by MIchaelMainer — CHANGES_REQUESTED** [review]

File: msgraph-sdk-raptor-compiler-lib/MicrosoftGraphCSharpCompiler.cs [hunk 2]
```diff
@@ -276,7 +278,7 @@ private async Task<string> GetATokenForGraph(string[] scopes)
             Assembly assembly = null;
 
             using MemoryStream memoryStream = new MemoryStream();
-            EmitResult emitResult = compilation.Emit(memoryStream);
+            EmitResult emitResult = compilation.Emit(memoryStream, options: new EmitOptions(metadataOnly: false, DebugInformationFormat.Embedded));
```
Comment (zengin) [inline msgraph-sdk-raptor-compiler-lib/MicrosoftGraphCSharpCompiler.cs:281]: Q: how do we consume this debug information?

Comment (georgend) [reply msgraph-sdk-raptor-compiler-lib/MicrosoftGraphCSharpCompiler.cs]: Step in debugging of the actual snippet during test debugging sessions.

Comment (zengin) [reply msgraph-sdk-raptor-compiler-lib/MicrosoftGraphCSharpCompiler.cs]: Ah, cool.

**Review by zengin — APPROVED** [review]

**Pull request merged**